pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator_helper.go

```diff
@@ -62,6 +69,11 @@ func (c *K8sOrchestrator) updatePVCAnnotations(ctx context.Context, volumeID str
 
 		pvcObj, err := c.informerManager.GetPVCLister().PersistentVolumeClaims(pvcNamespace).Get(pvcName)
 		if err != nil {
+			if apierrors.IsNotFound(err) {
```

Comment:
You dont have to do this if `getPVCAnnotations` returns `PVCNotFound` error.

Reply:
I think we should keep this check. Consider this:
`ClearFakeAttach` first calls `getPVCAnnotations` to check if a volume is fake attached, then `updatePVCAnnotations` to clear that annotation.
So if a fake attached PVC and corresponding pod are getting deleted, it may happen that `getPVCAnnotations` returns the annotations but just before `updatePVCAnnotations` call, PVC gets deleted from api server.
So it at all there's any race condition, we should return `NotFound` error from here and handle it in  `ClearFakeAttach`.

Reply:
Makes sense.